src/geo/transform.js

```diff
@@ -225,13 +226,13 @@ class Transform {
         this._updateCameraOnTerrain();
         this._calcMatrices();
     }
-    updateElevation(constrainCameraOverTerrain: boolean) { // On render, no need for higher granularity on update reasons.
+    updateElevation(constrainCameraOverTerrain: boolean, isDragging: boolean) { // On render, no need for higher granularity on update reasons.
```

Comment:
Not your work, but I'm confused as to what this comment means!

Reply:
I don't know either. I went back to the original PR, but didn't connect it. Maybe I can remove it?

Yes good idea - definitely can have a default value